core/embed/trezorhal/stm32f4/secret.c

```diff
@@ -98,6 +98,10 @@ secbool secret_optiga_get(uint8_t dest[SECRET_OPTIGA_KEY_LEN]) {
   return secret_read(dest, SECRET_OPTIGA_KEY_OFFSET, SECRET_OPTIGA_KEY_LEN);
 }
 
+secbool secret_optiga_present(void) { return secret_wiped(); }
```

Comment:
This function should return inverted value of `secret_wiped()`.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/130162bfe30316f3605852066170800b2ad83b12